charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -17,6 +17,13 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
+  - security.gardener.cloud
+  resources:
+  - credentialsbindings
+  - workloadidentities
+  verbs:
+  - '*'
```

Comment:
If it is done this way, it will allow end users to send `create workloadidentity/token` request, this grant users with direct access to the infrastructure API, which for some users might mean privilege escalation as they might not be granted with access to the infrastructure.

Reply:
Should we also add a comment so we do not forget about this detail? 🤔

Reply:
We could add a comment to make this obvious as someone might decide to "refactor" this tomorrow and regress it.